ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/InternalValidationResult.java

```diff
@@ -23,8 +23,11 @@
       InternalValidationResult.create(ValidationResultCode.ACCEPT);
   public static InternalValidationResult IGNORE =
       InternalValidationResult.create(ValidationResultCode.IGNORE);
+
+  @Deprecated
   public static InternalValidationResult REJECT =
       InternalValidationResult.create(ValidationResultCode.REJECT);
```

Comment:
I'd suggest inlining this rather than deprecating.  We have so much stuff deprecated now that it's not a particularly helpful signal so may as well just clean it up even if not everywhere is passing a message yet.

Reply:
Or actually I'd provide a generic error message here, then inline it.  That way everywhere that calls it will wind up with a generic "Failed validation" error message instead of an empty optional which is obviously unhelpful but doesn't wind up being ignored because it's an empty optional like happened in this case.